### ISSUE TITLE
Spin off widget/cocoa and */themes/osx into widget/tiger and */themes/tiger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [classilla/tenfourfox#89]
```
We need to make our own widget/src/tiger because of all the changes that 
Mozilla keeps making, especially when 10.5 support ends (see issue 19, issue 
88, others).
```

Original issue reported on code.google.com by `classi...@floodgap.com` on 29 Sep 2011 at 12:38
* Blocking: #173, #182, #164, #16
* Blocked on: #83

**Comment by classilla** [comment]
Wontfix. We are now at 45 and we're moving from source parity anyway so there is no benefit.
